qa/suites/crimson-rados/basic/tasks/rados_python.yaml

```diff
@@ -17,4 +17,4 @@ tasks:
     timeout: 1h
     clients:
       client.0:
-        - rados/test_python.sh -m 'not (wait or tier or ec)'
+        - rados/test_python.sh -m 'not (tier or ec)'
```

Comment:
This seems to be an unrelated change -- this test doesn't fall under RBD.

Reply:
Moved out to include only rbd api related tests, we would still need to verify which tests are ok and which still require missing support. Moving to draft for now.